components/kyma-environment-broker/internal/cls/utils.go

```diff
@@ -0,0 +1,34 @@
+package cls
+
+import (
+	"encoding/json"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage"
+	"github.com/pkg/errors"
+)
+
+func EncryptOverrides(secretKey string, overrides *ClsOverrides) (string, error) {
```

Comment:
These functions can also be a part of `overrides.go` file.

Reply:
done.